pkg/ebpf/bytecode/runtime/runtime_asset.go

```diff
@@ -40,6 +41,11 @@ type CompiledOutput interface {
 type RuntimeAsset struct {
 	filename string
 	hash     string
+
+	// Telemetry
+	compilerEnabled     int64
+	compilationSuccess  int64
```

Comment:
There is more than one `RuntimeAsset` object in the system-probe, should we not collect stats for all of them?

Reply:
Or, specific stats for each `RuntimeAsset`? We can either use prefixes in the stats name, or list compilation stats for each asset in its own sub-section.

Reply:
We are quickly going to have more `RuntimeAsset` objects (conntrack likely being one), so we should setup a schema that supports that.

Reply:
I think listing the stats for each asset in its own sub-section of the `RuntimeAsset` struct & having designated functions to retrieve stats for each type of asset (ie `GetCompilerStats`, `GetConntrackStats`, etc) is a sufficient way to handle this, at least for now. If we find when adding more assets that this becomes overwhelming, we can always revisit, but idk if we should make it too complicated before it needs to be

Reply:
At this time, as far as I know the only other runtime asset we have is `RuntimeSecurity`, which is also a runtime compiler, so the telemetry metrics added here apply to that asset as well

Reply:
On second thought, after taking a look at the conntrack telemetry, I decided it was worth it to take the time now to create a schema for telemetry which will support different types of telemetry for different types of runtime assets